Channel/DoctrineChannel.php

```diff
@@ -33,7 +33,7 @@ public function __construct(EntityManager $manager)
      */
     public function supports($recipient)
     {
-        if (is_object($recipient) && $recipient instanceof DoctrineRecipientInterface) {
```

Comment:
You could just `return $recipient instanceof IdentifierRecipientInterface;`

Reply:
Agreed with this